app/views/components/_dropdown.html.erb

```diff
@@ -0,0 +1,21 @@
+<%
+  id ||= "image-meta-#{SecureRandom.hex(4)}"
+  items ||= []
+  data_attributes ||= {}
+  data_attributes[:module] = 'dropdown'
+%>
+
+<%= tag.details class: "app-c-dropdown", data: data_attributes do %>
```

Comment:
W3C indicates this doesn't work on IE/Edge. Is that a problem?

Reply:
We have a details polyfill covering for IE/Edge.